packages/agoric-cli/src/commands/vaults.js

```diff
@@ -2,28 +2,61 @@
 /* eslint-disable @jessie.js/no-nested-await */
 // @ts-check
 /* eslint-disable func-names */
-/* global fetch */
+/* global fetch, process */
+import { vstorageKeySpecToPath } from '@agoric/casting';
 import { Command } from 'commander';
+import { normalizeAddressWithOptions } from '../lib/chain.js';
 import { makeRpcUtils } from '../lib/rpc.js';
 import { makeOpenSpendAction } from '../lib/vaults.js';
-import { outputAction } from '../lib/wallet.js';
+import { getCurrent, outputAction } from '../lib/wallet.js';
 
 /** @typedef {import('@agoric/smart-wallet/src/offers').OfferSpec} OfferSpec */
 /** @typedef {import('@agoric/smart-wallet/src/offers').OfferStatus} OfferStatus */
 /** @typedef {import('@agoric/smart-wallet/src/smartWallet').BridgeAction} BridgeAction */
 
-const {
-  vstorage,
-  fromBoard: _fromBoard,
-  agoricNames,
-} = await makeRpcUtils({ fetch });
+const { vstorage, fromBoard, agoricNames } = await makeRpcUtils({ fetch });
 
 /**
  *
  * @param {import('anylogger').Logger} logger
  */
 export const makeVaultsCommand = async logger => {
-  const vaults = new Command('vaults').description('Vault Factory commands');
+  const vaults = new Command('vaults')
+    .description('Vault Factory commands')
+    .option('--home [dir]', 'agd application home directory')
+    .option(
+      '--keyring-backend [os|file|test]',
+      'keyring\'s backend (os|file|test) (default "os")',
+    );
+
+  const normalizeAddress = literalOrName =>
+    normalizeAddressWithOptions(literalOrName, vaults.opts());
+
+  vaults
+    .command('list')
+    .description(
+      'list vaults ever owned by the address (as path that can be followed)',
+    )
+    .requiredOption(
+      '--from <address>',
+      'wallet address literal or name',
+      normalizeAddress,
+    )
+    .action(async function () {
+      // @ts-expect-error this implicit any
+      const opts = this.opts();
```

Comment:
is `opts` available as an argument to the action function?

Reply:
looks like somewhere along the way they added that https://www.npmjs.com/package/commander#action-handler

I'll tackle that in upgrading Commander to v10